assets/scss/btn.scss

```diff
@@ -0,0 +1,46 @@
+.btn {
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  cursor: pointer;
+  display: inline-block;
+  padding: .375rem .75rem;
+  transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+
+  &:active, &:focus, &:hover {
+    text-decoration: none;
+  }
+
+  &:hover {
+    background-color: #f9f9f9;
+  }
+}
+
+.btn--minfraud {
+  background-color: var(--mm-color-minfraud);
+  border-color: var(--mm-color-minfraud);
+  color: #fff;
+
+  &:hover {
+    background-color: #8a3899;
+  }
+}
+
+.btn--geoip {
+  background-color: var(--mm-color-geoip);
+  border-color: var(--mm-color-geoip);
+  color: #fff;
+
+  &:hover {
+    background-color: #217e71;
+  }
+}
+
+.btn--primary {
+  background-color: #4877bb;
+  border-color: #4877bb;
```

Comment:
Good call.  I'll try to come up with a different name so that we don't end up with `mm-color-[some other adjective]-blue`.  Or we could go the tailwind route and start adding numbers like `mm-color-blue-500`.